### PR TITLE
bokeh views deployment

### PR DESCRIPTION
**Pull request by jhoblitt**
Support qa-dashboard deployment post DM-6992.

### CODE REVIEW

File: squash/dashboard/views.py [hunk 1]
```diff
@@ -71,6 +78,6 @@ class MetricsView(ListView):
 
     def get_context_data(self, **kwargs):
         context = super(MetricsView, self).get_context_data(**kwargs)
-        bokeh_script = autoload_server(None, app_path="/metrics")
+        bokeh_script = autoload_server(None, app_path="/metrics", url=bokeh_url)
```
Comment (afausti) [inline squash/dashboard/views.py]: Sorry @jhoblitt I was careless in removing bokeh_url - though it was not necessary in this new django/bokeh integration method.

File: squash/dashboard/templates/base.html [hunk 2]
```diff
@@ -51,6 +51,7 @@
                 <p>&copy; Copyright {% now 'Y' %}, AURA/LSST</p>
             </div>
         </footer>
+        <script src="{% static 'js/jquery.min.js' %}"></script>
```
Comment (afausti) [inline squash/dashboard/templates/base.html]: @jhoblitt, so that fix the bootstrap warning thanks

**Comment by afausti** [comment]
All good @jhoblitt sorry for the extra work adding these things back.

**Comment by jhoblitt** [comment]
Thanks for the fast review!

**Pull request merged**